src/main/java/seedu/address/logic/parser/InvalidLoginParser.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.LoginCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+//@@author: IcedCoffeeBoy
+
+/**
+ * Parses user input.
+ */
+public class InvalidLoginParser implements Parser<LoginCommand> {
+
+    private static final String MESSAGE_INVALID_LOGIN = "User already login as %s"
+            + "\nPlease logout to login again"
+            + "\nExample: logout";
+
+    /**
+     * Parses user input into command for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public LoginCommand parse(String identity) throws ParseException {
+        throw new ParseException(String.format(MESSAGE_INVALID_LOGIN, identity));
```

Comment:
why is there a separate parser class which first does not parse, and second only throws a ParseException. the throwing of ParseException can be added to the Manager/Employee parser without the need of a separate class. having the class called Parser can be misleading as it does not work as a parser.

Reply:
how you you suggest to do throw the exception without breaking the OOP?

Reply:
i can throw exception directly

Reply:
will throwing exception directly break OOP?